connect.go

```diff
@@ -66,6 +68,8 @@ type ConfigurationsUpdate struct {
 	Update      bool   `json:"update"`
 }
 
+var producersMap = make(ProducersMap)
```

Comment:
Please make this variable a member of the connection struct, for cases of multiple connections from a single application